lib/utils.js

```diff
@@ -458,7 +456,12 @@ export function moveElementAwayFromCollision(
   const compactH = compactType === "horizontal";
   // Compact vertically if not set to horizontal
   const compactV = compactType !== "horizontal";
-  const preventCollision = false; // we're already colliding
+  let preventCollision = false; // we're already colliding
```

Comment:
it can be done a little bit simpler:

```typescript
const preventCollision = collidesWith.static
```

and, I think, we should change this comment `we're already colliding` -> `we're already colliding (not for static items)` and remove comment `when preventCollision = false and element is static then we have RangeError: Maximum call stack size exceeded`, it's better to place this comment into the commit message

Reply:
I agree, thank you.